Source/engine/render/light_render.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "engine/point.hpp"
+
+namespace devilution {
+
+class Lightmap {
+public:
+	Lightmap(uint8_t *outBuffer);
```

Comment:
Single-argument constructors should be marked `explicit` https://abseil.io/tips/142
```suggestion
	explicit Lightmap(uint8_t *outBuffer);
```

Reply:
I think there is a way to further simplify / isolate this:
1. Pass `LightTables` here as an argument.
2. Pass everything needed for `BuildLightmap` as arguments here and calling `BuildLightmap` in the constructor (also things like `gnScreenWidth` should be arguments).